tests/dummy/app/routes/nodes/detail/draft-registrations/index.js

```diff
@@ -8,6 +8,8 @@ export default Ember.Route.extend({
     },
     setupController(controller, model) {
         this._super(controller, model);
+        var metaschemas = this.store.findAll('metaschema');
+        controller.set('metaschemas', metaschemas);
```

Comment:
Yes - good point.  I am currently going to set the controller property after promise resolved per your first suggestion.  Drafts are going to need a large overhaul at some point and this can go in at that time (I don't think drafts are the big priority right now). My main reason for including this fix in this PR was so drafts as a relationship on the node could be tested.